electrum/gui/qt/__init__.py

```diff
@@ -53,11 +53,18 @@
 
 
 try:
-    from . import icons_rc
+    import importlib
+    from pathlib import Path
+    path = str(Path(__file__).parent.joinpath("icons/icons_rc.py"))
+    spec = importlib.util.spec_from_file_location("icons_rc", path)
+    module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(module)
```

Comment:
By adding `__init__.py` into the icons submodule it would become a python package and could be imported using a simple `import` statement. Also, pyinstaller would then automatically pick up that file. If I leave this as is then the `*.spec` files would need some adjustment. I prefer the icons package solution. What do you think?

Reply:
@bauerj any opinion on this?

Reply:
Although a package containing only a single module looks wrong I think it does make sense in this context.